playground/playground.go

```diff
@@ -200,13 +201,26 @@ func main() {
 		}()
 
 		scope.Set("format", func() {
-			out, err := format.Source([]byte(scope.Get("code").String()))
-			if err != nil {
-				scope.Set("output", []Line{Line{"type": "err", "content": err.Error()}})
-				return
-			}
-			scope.Set("code", string(out))
-			scope.Set("output", []Line{})
+			go func() {
+				var out []byte
+				var err error
+				switch dom.GetWindow().Document().GetElementByID("imports").(*dom.HTMLInputElement).Checked {
```

Comment:
There may be a better angular way of accessing the value of a checkbox; I do not know what it is, so I just accessed the DOM directly.